examples/ota-requestor-app/efr32/args.gni

```diff
@@ -23,6 +23,9 @@ pw_assert_BACKEND = "$dir_pw_assert_log"
 chip_enable_openthread = true
 chip_openthread_ftd = false
 
+# Disable stack lock tracking
+chip_stack_lock_tracking = "None"
```

Comment:
Why?

Reply:
Yes indeed; below you added the lock/unlock why then it is disabled an explanation will be welcomed

Reply:
We don't plan on maitaining the example so getting it to work with stack locking isn't a priority for us.